core/src/main/java/org/testcontainers/containers/output/Slf4jLogConsumer.java

```diff
@@ -9,13 +9,19 @@
  */
 public class Slf4jLogConsumer implements Consumer<OutputFrame> {
     private final Logger logger;
+    private String prefix = "";
 
     public Slf4jLogConsumer(Logger logger) {
         this.logger = logger;
     }
 
+    public Slf4jLogConsumer setPrefix(String prefix) {
+        this.prefix = "["+prefix+"]";
```

Comment:
How about appending a space after the closing square bracket, and removing the first space from the `logger.info` pattern string? 

That way the log message won't have an unnecessary space (before the frame type) when the prefix hasn't been set.